lib/stop_times.js

```diff
@@ -78,7 +78,7 @@ const formatStopTimesRow = (s) => {
 		s.stop_headsign || null,
 		s.pickup_type ? pickupDropOffType(s.pickup_type) : null,
 		s.drop_off_type ? pickupDropOffType(s.drop_off_type) : null,
-		s.shape_dist_traveled ? parseInt(s.shape_dist_traveled) : null,
+		s.shape_dist_traveled ? parseFloat(s.shape_dist_traveled) : null,
```

Comment:
Actually I'm wondering if we should do any kind of parsing at all here... 🤔 – PostgreSQL will cast the strings from the CSV to a `REAL` anyways, right?